doc/sources/tutorials/pong.rst

```diff
@@ -430,6 +430,84 @@ pong.kv:
     :language: kv
     :linenos:
 
+Improve Bouncing
+---------------
+
+..container:: title
+
+    Just one bounce per collision
+   
+If you try to hit ball with upper or lower part of the paddle and moving, you'll notice that the bounce_ball is fired many times, causing undesired behavior.
+
+To solve that, we can add a bouncer controller, that enables and disables bouncing ability. In short, bouncing is disabled when paddle_collides ball, and gets re-enabled when is not colliding again.
```

Comment:
Understood pep8 and rename manager to controller. I didn't get the part of backticks. (i know what a backtick char is) but i cant see that in this context.

Reply:
Well, as I've written, you should highlight the interesting parts i.e. function/method names, etc:

> paddle_collides → `paddle_collides`

so that it's not just a block of text because people tend to overlook/ignore such stuff.

Reply:
Ohh, true. Il'l review all those notes and come back with clean PR. Thanks.